settings.gradle

```diff
@@ -5,4 +5,5 @@ include 'client'
 include 'server'
 include 'values'
 include 'users'
+include 'tools'
```

Comment:
The doclet code itself  should have been implemented in [this repository](https://github.com/SpineEventEngine/tools), not in a new module.